experimental/coref/README.md

```diff
@@ -0,0 +1,108 @@
+
+Before using this project:
+
+1. install spaCy with GPU support - see the [install widget](https://spacy.io/usage)
+2. run `pip install -r requirements.txt`
+3. modify `project.yml` to set your GPU ID and OntoNotes path (see [Data Preparation](#data-preparation) for details)
+
+After that you can just run `spacy project run all`.
+
+Note that during training you will probably see a warning like `Token indices
+sequence length is longer than ...`. This is a rare condition that
+`spacy-transformers` handles internally, and it's safe to ignore if it
+happens occasionally. For more details see [this
+thread](https://github.com/explosion/spaCy/discussions/9277#discussioncomment-1374226).
+
+## Data Preparation
+
+To use this project you need a copy of [OntoNotes](https://catalog.ldc.upenn.edu/LDC2013T19).
+
+If you have OntoNotes and have not worked with the CoNLL 2012 coreference annotations before, set `vars.ontonotes` in `project.yml` to the local path to OntoNotes. The top level directory should contain directories named `arabic`, `chinese`, `english`, and `ontology`. Then run the following command to prepare the coreference data:
+
+```
+spacy project run prep-conll-data
+```
+
+After that you can execute `spacy project run all`.
+
+If you already have CoNLL 2012 data prepared and concatenated into one file per split, you can specify the paths to the training, dev, and test files directly in `project.yml`, see the `vars` section. After doing so you can run `spacy project run all`.
+
+## Using the Trained Pipeline
+
+After training the pipeline (or downloading a pretrained version), you can load and use it like this:
+
+```
+import spacy
+nlp = spacy.load("training/coref")
+
+doc = nlp("John Smith called from New York, he says it's raining in the city.")
+# check the word clusters
+print("=== word clusters ===")
+word_clusters = [val for key, val in doc.spans.items() if key.startswith("coref_head")]
+for cluster in word_clusters:
+    print(cluster)
+# check the expanded clusters
+print("=== full clusters ===")
+full_clusters = [val for key, val in doc.spans.items() if key.startswith("coref_cluster")]
+for cluster in full_clusters:
+    print(cluster)
+```
+
+The prefixes used here are a user setting, so you can customize them for your
+own pipelines.
+
+
+## 📋 project.yml
+
+The [`project.yml`](project.yml) defines the data assets required by the
+project, as well as the available commands and workflows. For details, see the
+[spaCy projects documentation](https://spacy.io/usage/projects).
+
+### ⏯ Commands
+
+The following commands are defined by the project. They
+can be executed using [`spacy project run [name]`](https://spacy.io/api/cli#project-run).
+Commands are only re-run if their inputs have changed.
+
+| Command | Description |
+| --- | --- |
+| `prep-ontonotes-data` | Rehydrate the data using OntoNotes |
+| `prep-test-data` | Prepare minimal dataset for CI testing. Note this will overwrite train/dev/test data! |
```

Comment:
Users might be interpreting "test" as "test data" rather than "unit test" here, so I wonder whether there's anyway we can make it more clear to users that they shouldn't really call this. I'd also consider using the explicit term "unit test" somewhere, in addition to "CI".

So maybe something like `prep-artificial-unit-test-data` (clunky on purpose) ?

Reply:
Sounds good.